e2e-tests/playwright/specs/accessibility/channels/settings_dialog/display.spec.ts

```diff
@@ -0,0 +1,345 @@
+        await pw.toBeFocusedWithFocusVisible(settingsModal.container);
+
+        // # Press Tab and verify focus is on Close button
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(settingsModal.closeButton);
+
+        // # Click on Display tab to open it
+        await settingsModal.displayTab.click();
+
+        // * Display Settings panel should be visible
+        await expect(displaySettings.container).toBeVisible();
+
+        // # Press Tab to move focus to first edit button (Theme)
+        await page.keyboard.press('Tab');
+        await pw.toBeFocusedWithFocusVisible(displaySettings.themeEditButton);
+
+        // # Continue pressing Tab until we reach Timezone button (skipping intermediate sections)
+        // There are several edit buttons between Theme and Timezone
+        for (let i = 0; i < 5; i++) {
+            await page.keyboard.press('Tab');
+        }
+        await pw.toBeFocusedWithFocusVisible(displaySettings.timezoneEditButton);
+
+        // # Continue pressing Tab until we reach Language button (skipping intermediate sections)
+        // There are several edit buttons between Timezone and Language
+        for (let i = 0; i < 8; i++) {
+            await page.keyboard.press('Tab');
+        }
+        await pw.toBeFocusedWithFocusVisible(displaySettings.languageEditButton);
+    },
+);
+
+/**
+ * @objective Verify Display settings panel passes accessibility scan and matches aria-snapshot
+ */
+test(
+    'accessibility scan and aria-snapshot of Display settings panel',
+    {tag: ['@accessibility', '@settings', '@display_settings', '@snapshots']},
+    async ({pw, axe}) => {
+        // # Create and sign in a new user
+        const {user} = await pw.initSetup();
+
+        // # Log in a user in new browser context
+        const {page, channelsPage} = await pw.testBrowser.login(user);
+        const globalHeader = channelsPage.globalHeader;
+        const settingsModal = channelsPage.settingsModal;
+
+        // # Visit a default channel page
+        await channelsPage.goto();
+        await channelsPage.toBeVisible();
+
+        // # Set focus to Settings button and press Enter
+        await globalHeader.settingsButton.focus();
+        await page.keyboard.press('Enter');
+
+        // * Settings modal should be visible
+        await expect(settingsModal.container).toBeVisible();
+
+        // # Open Display tab
+        await settingsModal.displayTab.click();
+
+        // * Display Settings panel should be visible
+        await expect(settingsModal.displaySettings.container).toBeVisible();
+
+        // * Verify aria snapshot of Display settings tab
+        await expect(settingsModal.displaySettings.container).toMatchAriaSnapshot(`
+          - tabpanel "display":
+            - heading "Display Settings" [level=3]
+            - heading "Theme" [level=4]
+            - button "Theme Edit"
+            - text: /.*/
```

Comment:
This should be replaced by the actual text presented in the UI. Same with other fields using `text: /.*/`